### PR TITLE
Bug 1719733: Export and import scripts for kibana objects

### PR DESCRIPTION
**Pull request by jcantrill**
This PR provides scripts to export and import a user's Kibana objects as requested to resolve: https://bugzilla.redhat.com/show_bug.cgi?id=1719733#c19

### CODE REVIEW

File: elasticsearch/README.md [hunk 1]
```diff
@@ -35,6 +35,21 @@ Example:
...
+```
+$ oc exec -c elasticsearch $POD -- export-kibana-objects admin2
+```
+### import-kibana-objects
+Import kibana objects from a JSON search response produced by `export-kibana-objects` to the kibana index in the
+input file.
+
+Example:
+```
+$ oc exec -i -c elasticsearch $POD -- import-kibana-objects kib.json
```
Comment (richm) [inline elasticsearch/README.md]: should this be
```
cat kib.json | oc exec -i -c elasticsearch $POD -- import-kibana-objects
```
?

File: elasticsearch/README.md [hunk 2]
```diff
@@ -35,6 +35,21 @@ Example:
 ```
 $ oc exec -c elasticsearch $POD -- allocate-stale-primary .kibana.02c55f18a892b365bcd1802db9e5c9df39c04674
 ```
+### export-kibana-objects
+Export kibana objects for a given user from their kibana index as a JSON search response
+
+Example:
+```
+$ oc exec -c elasticsearch $POD -- export-kibana-objects admin2
```
Comment (richm) [inline elasticsearch/README.md]: should you redirect the output e.g. `.... > kib.json` ?

File: elasticsearch/README.md (hunk 1 above)
Comment (jcantrill) [reply elasticsearch/README.md]: no.  the `-i` allows you to stream `CMD="oc exec -i -c elasticsearch $pod -- es_util" ./import-kibana-objects /home/centos/cluster-logging-tools/scripts/kib.json`

Comment (richm) [reply elasticsearch/README.md]: I don't understand.  So `import-kibana-objects` can be used both inside and outside of the container?

File: elasticsearch/README.md (hunk 2 above)
Comment (jcantrill) [reply elasticsearch/README.md]: yes.

File: elasticsearch/README.md (hunk 1 above)
Comment (jcantrill) [reply elasticsearch/README.md]: Updated and tested script reads from piped, stdin, updated README

**Pull request merged**